apps/mark-scan/frontend/src/app.tsx

```diff
@@ -81,8 +84,21 @@ export function App({
                 api={uiStringsApi}
                 disabled={!enableStringTranslation}
               >
-                <AppRoot hardware={hardware} reload={reload} logger={logger} />
-                <SessionTimeLimitTracker />
+                <ThemeConsumer>
```

Comment:
opt: heads-up that there's a `useCurrentTheme()` hook you could use that might make this code a little cleaner: https://github.com/votingworks/vxsuite/blob/main/libs/ui/src/hooks/use_current_theme.tsx